src/data/data.js

```diff
@@ -130,68 +82,27 @@ const pageData = {
           {
             src: '',
             alt: 'Piscina',
-            name: 'Catalina López',
+            component: 'Catalina López',
```

Comment:
This property is not referencing a component, but a data used inside one. So, this should not be altered.

Reply:
Done